src/WebpOptimizer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ImageCompression;
+
+use Exception;
+use WebPConvert\WebPConvert;
+
+class WebpOptimizer implements ImageOptimizerInterface
+{
+    const WEBP_EXT = 'webp';
+
+    public function optimizeImage(string $pathToImage, string $pathToOutput = null) : bool
+    {
+        try {
+            $pathToOutput = $this->getWebpPath($pathToOutput);
```

Comment:
Puede ser que acá sea asi?
```suggestion
            $pathToOutput = $this->getWebpPath($pathToImage);
```

Reply:
el tema es mas para garantizar que la extension sea `.webp`.
el metodo este esta bajo una Interface que ya utilizamos actualmente en el core y le pasamos ambos valores. Yo lo dejaría asi, que opinas?

Reply:
Perdon, entendi mal. Ahi subi esa parte